File: 백준/Silver/1003. 피보나치 함수/피보나치 함수.rs
use std::io::{BufReader,BufRead,BufWriter,stdin,Write,stdout};
fn main(){
    let mut reader= BufReader::new(stdin().lock());
    let mut writer= BufWriter::new(stdout().lock());
    let mut input = String::new();
    reader.read_line(&mut input).unwrap();
  let t: i32 = input.trim().parse().unwrap();

    for _ in 0..t {
        input.clear();
        reader.read_line(&mut input).unwrap();
        let n: i32 = input.trim().parse().unwrap();
        
        let mut zero = 1;
        let mut one = 0;

        for _ in 0..n {
            let temp = zero;
            zero = one;
            one = zero + temp;
        }

        writeln!(writer,"{} {}", zero, one).unwrap();
    }
    writer.flush().unwrap();
}

File: 백준/Silver/1003. 피보나치 함수/피보나치 함수.py
import sys

t = int(sys.stdin.readline())

for _ in range(0,t) :
    n = int(sys.stdin.readline())
    zero=1
    one=0

    for _ in range(0,n):
        temp=zero
        zero=one
        one=zero+temp
    
    sys.stdout.write(f"{zero} {one}\n")

sys.stdout.flush() 